tests/homepage/ManageAddon.test.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import ManageApps from '../../app/js/components/homePage/manageApps/manageApps';
+import BreadCrumbComponent from '../../app/js/components/breadCrumb/breadCrumbComponent';
+import AddAddon from '../../app/js/components/homePage/manageApps/AddAddon';
```

Comment:
Any reason you have this import here since it wasn't used anywhere?

Reply:
Good question @kingisaac95 
@andela-jomadoye can you address it in a followup pull request?